### DISCUSSION TITLE
请教一下，要如何训练底模呢？

### DISCUSSION THREAD

**Discussion by zxw1989kl** [fishaudio/Bert-VITS2#287] [General]
我想重新训练24k的底模，用了15人300小时数据，重头训练，训练了G_14000.pth，测试合成的音都不对。请问要如何训练底模呢？

**Comment by Stardust-minus** [comment]
底模需要分3-4个stage训练，中途有非常多的tricks，不建议任何人重训底模

**Reply by Shuigs18** [reply]
大佬能分享下训练过程吗😂

**Reply by hopingZ** [reply]
减佬求教学！在训一个 16k 英文版本，拿了 LJS 预训练，收是收敛了，但一直达不到那么好的效果。可以提示一下嘛 ~
